COMPATIBILITY.md

```diff
@@ -119,6 +119,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [X] `buildtriggerbadge`: supported as of 2.2
 - [X] `radiatorview`: supported as of 1.25
 - [ ] `chucknorris`: [JENKINS-32594](https://issues.jenkins-ci.org/browse/JENKINS-32594)
+- [ ] `Zephyr` : [JENKINS-32801](https://issues.jenkins-ci.org/browse/JENKINS-32801)
```

Comment:
And is probably in the wrong section. You meant to put this under _Build steps and post-build actions_.